### PR TITLE
Adds a basic sscan

### PR DESCRIPTION
**Pull request by robertmarney**
This is imperfect as we are not implementing the actual scan options, however for the purposes of mocking it should be sufficient for most uses cases.

This allows the use of tagged collections in Laravel ^8.76

### CODE REVIEW

File: src/M6Web/Component/RedisMock/RedisMock.php [hunk 1]
```diff
@@ -379,6 +379,16 @@ public function smembers($key)
...
     }
+    
+    public function sscan($key, $cursor = '0', array $options = [])
+    {
+        if (!isset(self::$dataValues[$this->storage][$key]) || $this->deleteOnTtlExpired($key)) {
+            return $this->returnPipedInfo([$cursor,[]]);
+        }
+        // For the purposes of this mock we returning a simple chunk of all data similer to smembers
+        $chunk =  $this->returnPipedInfo(self::$dataValues[$this->storage][$key]);
+        return [$cursor, $chunk];
```
[the remark this answers is not in the document]
Comment (robertmarney) [reply src/M6Web/Component/RedisMock/RedisMock.php:390]: Apologies for the slow response. I am not following.

[the remark this answers is not in the document]
Comment (lnahiro) [reply src/M6Web/Component/RedisMock/RedisMock.php:390]: I'm wondering since all data are returned, the cursor return by the function should be 0.

**Comment by thirsch** [comment]
Any update @robertmarney? We are facing the same issue since we've upgraded from Laravel 6 to 9 a couple of days ago.

**Comment by omansour** [comment]
see #107 seems better

**Pull request closed without merging**